packages/driver/cypress/integration/commands/actions/attachFile_spec.js

```diff
@@ -106,6 +104,16 @@ describe('src/cy/commands/actions/attachFile', () => {
       })
     })
 
+    it('attaches files to an input when dragged to it', () => {
```

Comment:
Is this test a duplicate of the next test? Do we need both?

Reply:
Ah, yep. Removing duplicate test.